homeworks/olexandr.reshetnyk_reshetnyk/homework_5/task_2.js

```diff
@@ -0,0 +1,20 @@
+const classes = ['header', 'menu', 'menu_item', 'tabs',
+    'tab_item', 'menu', 'link', 'tabs', 'tab_item', 'menu',
+    'menu_item', 'menu', 'menu_item'];
+
+function getAmountsOfOccurrencesOfElements(array) {
+    const resObj = {};
+
+    function occurencesAmount(neededElem) {
+        return array.filter(elem => elem === neededElem).length;
+    }
+
+    array.forEach((elem) => {
+        if (!(elem in resObj)) {
+            resObj[elem] = occurencesAmount(elem);
+        }
+    });
+
+    return resObj;
+}
```

Comment:
looks like working solution, but why loop over same array multiple times, if you could simply `reduce()` it once and get much more readable result